extlinks/settings/staging.py

```diff
@@ -0,0 +1,17 @@
+from .base import *
+
+DEBUG = False
+
+ALLOWED_HOSTS += ["wikilink-staging.wmflabs.org"]
```

Comment:
We should have as few allowed hosts as possible.
```suggestion
ALLOWED_HOSTS = ["wikilink-staging.wmflabs.org"]
```

Reply:
That makes sense!